spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -358,14 +358,14 @@ install the Spring Boot CLI is:
 
 
 [[getting-started-cli-command-line-completion]]
-==== Command-line completion
-Spring Boot CLI ships with scripts that provide command completion for
-http://en.wikipedia.org/wiki/Bash_%28Unix_shell%29[BASH] and
+==== Command-line Completion
+The Spring Boot CLI includes scripts that provide command completion for
+the http://en.wikipedia.org/wiki/Bash_%28Unix_shell%29[BASH] and
 http://en.wikipedia.org/wiki/Zsh[zsh] shells. You can `source` the script (also named
-`spring`) in any shell, or put it in your personal or system-wide bash completion
-initialization. On a Debian system the system-wide scripts are in `/shell-completion/bash`
+`spring`) in any shell or put it in your personal or system-wide bash completion
+initialization. On a Debian system, the system-wide scripts are in `/shell-completion/bash`
 and all scripts in that directory are executed when a new shell starts. To run the script
-manually, e.g. if you have installed using SDKMAN!
+manually if you have installed using SDKMAN!, use the following commands:
```

Comment:
I think SDKMAN! needs to be an example here. There are others reasons why you may want to install the script manually, not just if you installed the CLI with SDKMAN!

Reply:
OK. I made that change.